ui/cypress/e2e/explorer.test.ts

```diff
@@ -351,6 +351,10 @@ describe('DataExplorer', () => {
   describe('raw script editing', () => {
     beforeEach(() => {
       cy.getByTestID('switch-to-script-editor').click()
+
+      // explicitly wait, to give the CodeMirror editor a chance to render
+      // may not be necessary when we switch to Monaco
```

Comment:
We can explicitly wait on the CodeMirror editor to render by waiting for the `.CodeMirror` class see `'imports the appropriate packages to build a query'` test for an example

Reply:
@121watts i could not get this test to work without the wait. it would always fail on ci (but pass locally). to get this out, do you mind if I keep the `wait` in there and then look at this (and some other tests) next week?